features/steps/flaw_detail.py

```diff
@@ -551,8 +551,10 @@ def step_impl(context):
     flaw_detail_page.check_value_exist(context.ps_component)
     context.browser.quit()
 
-@when("I delete an affect of the flaw")
-def step_impl(context):
+def click_affect_delete_btn(context):
```

Comment:
Better put this function into `FlawDetailPage` class

Reply:
Updated.